provisioning_templates/PXEGrub2/kickstart_default_pxegrub2.erb

```diff
@@ -26,14 +26,26 @@ set default=0
 set timeout=<%= host_param('loader_timeout') || 10 %>
 
 menuentry '<%= template_name %>' {
-  <%= linuxcmd %> <%= @kernel %> ks=<%= foreman_url('provision') %> <%= pxe_kernel_options %> <%= snippet("kickstart_kernel_options").strip %> BOOTIF=01-$net_default_mac
+  <%= linuxcmd %> <%= @kernel %> ks=<%= foreman_url('provision') %> <%= pxe_kernel_options %> <%= snippet("kickstart_kernel_options").strip %>
   <%= initrdcmd %> <%= @initrd %>
 }
 
-# Workaround for https://bugzilla.redhat.com/show_bug.cgi?id=1811561
-menuentry '<%= template_name %> HTTP Boot' {
-  <%= linuxcmd %> /httpboot/<%= @kernel %> ks=<%= foreman_url('provision') %> <%= pxe_kernel_options %> <%= snippet("kickstart_kernel_options").strip %> BOOTIF=01-$net_default_mac
-  <%= initrdcmd %> /httpboot/<%= @initrd %>
+<%
+if @host.provision_interface.subnet.httpboot
+  proxy_http_port = @host.provision_interface.subnet.httpboot.setting(:HTTPBoot, "http_port")
+  proxy_https_port = @host.provision_interface.subnet.httpboot.setting(:HTTPBoot, "https_port")
```

Comment:
I'm getting
```
undefined method '#setting' for SmartProxy::Jail (SmartProxy)
```

Is there a PR that exposes `setting` on smart proxy jail t hat I missed?

Reply:
Damn, I had to turn it off on my instance and forgot this one. Lemme update the other PR then.